pytest_pyodide/runner.py

```diff
@@ -610,7 +607,20 @@ def run_js_inner(self, code, check_code):
 
         cmd_id = str(uuid4())
         self.p.sendline(cmd_id)
-        self.p.sendline(wrapped)
+        # split lines into shorter buffers
+        # because some ttys don't like long
```

Comment:
Could you add a test for passing a very long string? I think we don't have a testcase for this.

Reply:
I think run_in_pyodide tests cover this - they send a 5kb string.